docker-monolith/infra/ansible/roles/gitlab_omnibus/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+env: local
+gitlab_ci_external_ip: 0.0.0.0
```

Comment:
You renamed the role but forget to rename variable prefixes. There (and other places) should be `gitlab_omnibus_external_ip`